lib/UndefinedValue.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for undefined values
+ *
+ * @extends Error
+ * @class Errors.UndefinedValue
+ * */
+class UndefinedValue extends Error{
+    /**
+   * @example
+   *
+   *if(!importantPassword){
+       const err = new Errors.Undefined;
+       err.setMessage(
+           'importantPassword',
+           importantPassword
+       );
+       throw err;
+   }
+   *
+   * @example
+   *
+   * ```sh
+    git/node-error-classes/example/undefinedValue.js:14
+       throw err;
+       ^
+
+    Undefined: 'string'
+
+   * ```
+   *
+   * @method Errors.UndefinedValue.setMessage
+   * @param  {Any} variable      varible
+   * @returns {String}    compiled error message
+   */
+    setMessage(variable){
+        this.name = 'Undefined';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/RIAEvangelist/node-error-classes/file/1941079327/issues/source?bid=3073169&fileBranchId=3104106#l39)